mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -1,13 +1,19 @@
 CLASS net/minecraft/class_127 net/minecraft/entity/LivingEntity
 	FIELD field_1009 maxHealth I
 	FIELD field_1019 texture Ljava/lang/String;
+	FIELD field_1024 scoreAmount I
+	FIELD field_1029 sideways F
 	FIELD field_1031 jumping Z
 	FIELD field_1033 movementSpeed F
 	FIELD field_1036 health I
 	FIELD field_1038 hurtTime I
 	FIELD field_1041 deathTime I
 	FIELD field_1042 attackCooldown I
+	FIELD field_1045 dead Z
+	FIELD field_1051 bodyTrackingIncrements I
+		COMMENT Used for lerp in the tickMovement method
 	FIELD field_1059 despawnCounter I
+	FIELD field_1060 forward F
```

Comment:
`forward` should be `forwardSpeed`
https://maven.fabricmc.net/docs/yarn-1.21+build.6/net/minecraft/entity/LivingEntity.html#forwardSpeed